asmcli/lib/checks.sh

```diff
@@ -242,7 +242,7 @@ is_service_mesh_feature_enabled() {
   local RESPONSE
   RESPONSE="$(run_command gcloud beta container hub mesh describe --project="${FLEET_ID}")"
 
-  if [[ "$(echo "${RESPONSE}" | jq -r '.featureState.lifecycleState')" != "ENABLED" ]]; then
+  if [[ "$(echo "${RESPONSE}" | jq -r '.featureState.lifecycleState' 2>/dev/null)" != "ENABLED" ]]; then
```

Comment:
Is this change intentional? Seems like unrelated.

Reply:
yes this is intentional. I noticed something during testing so I changed it. The error will show up in the log which is not desired.